test_rclcpp/test/test_publisher.cpp

```diff
@@ -31,52 +34,43 @@
 
 // Short test for the const reference publish signature.
 TEST(CLASSNAME(test_publisher, RMW_IMPLEMENTATION), publish_with_const_reference) {
-  rclcpp::init(0, nullptr);
-  auto node = rclcpp::Node::make_shared("test_publisher");
-
-  auto publisher = node->create_publisher<test_rclcpp::msg::UInt32>("test_publisher", 10);
-
+  // topic name
```

Comment:
The comment doesn't contain more information then the following variable name.

Same below.